### DISCUSSION TITLE
Close my issue and prevent it from reopened

### DISCUSSION THREAD

**Discussion by tisonkun** [community/community#23030] [New to GitHub]
<p>Recently, I close a series of issues I created. However, it is soon reopened by repo owner/collaborator although I want to no longer receive any message from it nor see it in my open issue list.</p>
<p>I can’t forbid the people reopening the issue. It is quite annoying that I lost the control of an issue created by me.</p>
<p>Any solution to prevent a closed issue created by me from reopened?</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/tisonkun">@tisonkun</a>,</p>
<p>You can unsubscribe from the issue using the button in the right-hand column on the issue page</p>
